system-tests/driver-config/jenkins.sh

```diff
@@ -17,7 +17,7 @@ targets:
 
     env:
       PROXIED_CLUSTER_URL: http://127.0.0.1:4201
-      INSTALLER_URL: "${INSTALLER_URL}"
+      INSTALLER_URL: ${INSTALLER_URL}
```

Comment:
Those quotes shouldn't be a problem. You're sure that we've to remove them?

Reply:
Was an issue in my local setup, would break and complain that yml was malformed:
```yml
---
...
INSTALLER_URL: ""http://{blablabla}""
```

Reply:
even `""` shouldnt be an issue 🤔 can we add the quotes again or also remove them from the next three lines? #ocd

Reply:
I'll add them back but then if it stops working for me I'll be asking everyone to run the system-tests for me :trollface: